tracing/src/main/java/io/airlift/tracing/TracingModule.java

```diff
@@ -15,19 +16,27 @@
 public class TracingModule
         extends AbstractConfigurationAwareModule
 {
+    private final Module module;
     private final String serviceName;
     private final String serviceVersion;
 
-    public TracingModule(String serviceName, String serviceVersion)
+    public TracingModule(Module module, String serviceName, String serviceVersion)
```

Comment:
Typically module dependencies aren't that explicit.
For example you need to install JsonModule  and NodeModule modules when you want to use TracingModule.

If this `module` parameter is something special, give it a name.